src/restclient/backup/backupservice.ts

```diff
@@ -66,7 +66,12 @@ export class BackupService extends RestService {
 			return await this.httpClient.send<BackupResult>(BackupService.API_BACKUP_ENDPOINT, HttpClient.NO_PAYLOAD,
 					<HttpResponseParser<BackupResult>> {
 				deserialize(content: any): BackupResult {
-					return JSON.parse(content)['state'] as BackupResult;
+					const obj = JSON.parse(content);
+					let result = new BackupResult();
+					result.state = obj['state'];
+					result.result = obj['result'];
+					result.message = obj['message'];
+					return result;
```

Comment:
You can simply remove the ['state'] part if the response object uses the same structure.

`return JSON.parse(content) as BackupResult;`